src/API/OnboardingThemes.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * REST API Onboarding Themes Controller
+ *
+ * Handles requests to install and activate themes.
+ *
+ * @package WooCommerce Admin/API
+ */
+
+namespace Automattic\WooCommerce\Admin\API;
+
+use Automattic\WooCommerce\Admin\Features\Onboarding;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Onboarding Themes Controller.
+ *
+ * @package WooCommerce Admin/API
+ * @extends WC_REST_Data_Controller
+ */
+class OnboardingThemes extends \WC_REST_Data_Controller {
```

Comment:
Can we add some test coverage for these new endpoints? Perhaps at least some assertions around the permission callbacks.

Reply:
Good idea!  Added tests for theme install and activation as well as the previous plugin API while I was there.